deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -655,11 +656,9 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
                 cell = audioMessageCell
             }
         case DC_MSG_VCARD:
-            cell = tableView.dequeueReusableCell(withIdentifier: "text", for: indexPath) as? TextMessageCell ?? TextMessageCell()
-            message.text = "👤 Contact"
-            if let file = message.file, let vcards = dcContext.parseVcard(path: file), let vcard = vcards.first {
-                message.text = "👤 " + vcard.displayName
-            }
+            guard let contactCell = tableView.dequeueReusableCell(withIdentifier: ContactCardCell.reuseIdentifier, for: indexPath) as? ContactCardCell else { fatalError("WTF wrong cell??") }
```

Comment:
let's do the same here as for the other cases.

even if that should really not happen, some little forgiveness may be appreciated. there may be cornercases, races, bugs in ios in future versions, bugs in delta chat in future versions. crashing the app when opening a chat is really a bad thing for the user.

if we're really sure this never can't happen (i am not :) we should change all cases in another PR. 

```suggestion
            let contactCell = tableView.dequeueReusableCell(withIdentifier: ContactCardCell.reuseIdentifier, for: indexPath) as? ContactCardCell ?? ContactCardCell()
```